src/main/java/org/terasology/dialogs/DialogScreen.java

```diff
@@ -33,14 +34,17 @@
     public static final ResourceUrn ASSET_URI = new ResourceUrn("Dialogs:DialogScreen");
 
     private ColumnLayout responseButtons;
+    private ColumnLayout responseImage;
     private BrowserWidget browser;
     private final HTMLDocument emptyPage = new HTMLDocument(null);
 
     @Override
     public void initialise() {
 
         responseButtons = find("responseButtons", ColumnLayout.class);
+        responseImage = find("responseImage", ColumnLayout.class);
```

Comment:
The default screen for dialogs does not have a column layout with this id yet. This should also be added in this PR.

Reply:
I had placed it here! https://github.com/Terasology/LightAndShadow/pull/139/files#diff-1e404c9eec7c3fa6cb41356bdb37b0e9R53

Reply:
Yes, and that's why it is working with Dialogs in LightAndShadow. But what happens in another game mode that does not customize the layout for the dialog window? What happens if the `find(...)` method does not find a `ColumnLayout` with id `responseImage`?

Reply:
Oh! Yes, indeed, I placed it at the override. Then setting it at the original UI file within the dialogue module should solve that issue, right? 
I did also look search a bit to some other files to see it `responseButtons` in case I need to set the new column layout somewhere else too, but it seems there isn't another place to do so. So just the original UI file mentioned should do it.